package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "restbase",
-  "version": "0.19.4",
+  "version": "0.19.5",
```

Comment:
No version bump, here, please.

Reply:
This change involves code changes not just dev depency changes. Is there a document that outlines when version numbers should or should not be changed?

Reply:
Rolled back.